### PR TITLE
Add new member dialog to admin

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Closes https://github.com/datagouv/data.gouv.fr/issues/1336
Replaces https://github.com/datagouv/udata-front/pull/398

It also renames MembershipRequest to AdminMembershipRequest to align with the naming of the other admin components.

It moves the edit member logic to its own components.

It also slightly improve the multiselect style to better match Figma.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Add organization edit to new admin [#412](https://github.com/datagouv/udata-front/pull/412)
 - Fix markdown tables and use DSFR ones [#432](https://github.com/datagouv/udata-front/pull/432)
+- Add beta admin "add member" modale [#398](https://github.com/etalab/udata-front/pull/398)
```
Comment (maudetes) [inline CHANGELOG.md:7]:
```suggestion
- Add beta admin "add member" modale [#442](https://github.com/etalab/udata-front/pull/442)
```

File: udata_front/theme/gouvfr/assets/js/pages/Admin/Members/Members.vue [hunk 2]
```diff
@@ -14,12 +14,23 @@
       <li>
       </li>
     </Breadcrumb>
-    <h1 class="fr-h1 fr-mb-2w">{{ t("Members") }}</h1>
+    <div class="fr-grid-row fr-grid-row--gutters fr-grid-row--middle justify-center">
+      <div class="fr-col-12 fr-col-md">
+        <h1 class="fr-h1 fr-mb-2w">{{ t("Members") }}</h1>
+      </div>
+      <div class="fr-col-auto">
+        <AdminAddMemberButton
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/pages/Admin/Members/Members.vue:22]: I think it should be shown only if `isAdmin`.

File: udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    const user = { ...userSuggested, avatar_thumbnail: userSuggested.avatar_url ?? undefined };
+    return  {
+      value: user.id,
+      label: user.first_name + " " + user.last_name,
+      image: useUserAvatar(user, 32),
+    };
+  });
+}
+
+function closeModal() {
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue:139]: We may want to clear modal and multiselect values when closing.

File: udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                  </h1>
+                  <MultiSelect
+                    :placeholder="$t('Search a user')"
+                    :searchPlaceholder="$t('Search a user...')"
+                    suggestUrl="/users/suggest/"
+                    :values="userId"
+                    @change="(value: string) => userId = value"
+                    :allOption="$t('Select a user')"
+                    :addAllOption="false"
+                    :onSuggest="formatUsers"
```
Comment (maudetes) [inline udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue:40]: When adding a member with suggest, I have too many calls made to `/avatars/../32 endpoint`.
I guess something is wrong with the multi-select, let me know if you reproduce.

**Review by maudetes — COMMENTED** [review]
Thank you! :pray:

File: udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue (hunk 4 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue:40]: We have a max options shown arbitrarily set as 200 so we might want to reduce it to a saner value like 50

Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue:40]: Trying now with 30

Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue:40]: Changed to 20

**Review by maudetes — APPROVED** [review]
I think we're clear, thank you!

**Pull request merged**